test/new/db/cmd/cmd_pd

```diff
@@ -1838,7 +1838,7 @@ e asm.arch=x86
 e asm.bits=32
 e asm.bytes=false
 s 0x1000
-wz path"C:\\Program\n
```

Comment:
This is a `"` without the matching closing `"`, so it is an error in the new parser. Let's just quote everything for now, but later this will become `wz "path\"C:\\Program\n"`

Reply:
well.. use backslash then to escape that

Reply:
That's what I'm doing.